lib/pharos/autoload.rb

```diff
@@ -66,6 +66,8 @@ module Terraform
 
   module Configuration
     autoload :Host, 'pharos/configuration/host'
+    autoload :Route, 'pharos/configuration/route'
```

Comment:
These are not used directly outside of `Host`... should `Host` just require them?

Reply:
Both are used by GatherFacts and neither are struct attributes of the Configuration::Host